.nvmrc

```diff
@@ -1 +1 @@
-12.21.0
```

Comment:
:grey_question: 
WDYT of bumping/removing `nvm install 10` from  https://github.com/woocommerce/google-listings-and-ads/blob/trunk/.travis.yml#L60:L60
So we will be sure the same node version is used for building and testing

Reply:
Good catch, I made it into `nvm install` which would [automatically respect `.nvmrc`](https://github.com/nvm-sh/nvm#nvmrc), in https://github.com/woocommerce/google-listings-and-ads/pull/801/commits/13e53498a7e5917165ce61f620ea578c14beeff9.